integrations/shuffle.py

```diff
@@ -0,0 +1,173 @@
+
+    debug("# File location")
+    debug(alert_file_location)
+
+    # Load alert. Parse JSON object.
+    try:
+        with open(alert_file_location) as alert_file:
+            json_alert = json.load(alert_file)
+    except:
+        debug("# Alert file %s doesn't exist" % alert_file_location)
+
+    debug("# Processing alert")
+    try:
+        debug(json_alert)
+    except Exception as e:
+        debug("Failed getting json_alert %s" % e)
+        sys.exit(1)
+
+    debug("# Generating message")
+    msg: str = generate_msg(json_alert)
+
+    # Check if alert is skipped
+    if isinstance(msg, str):
+        if not msg:
+            return
+
+    debug(msg)
+
+    debug("# Sending message")
+
+    send_msg(msg, webhook)
+
+
+def debug(msg):
+    if debug_enabled:
+        msg = "{0}: {1}\n".format(now, msg)
+        print(msg)
+        f = open(log_file, "a")
+        f.write(msg)
+        f.close()
+
+
+# Skips container kills to stop self-recursion
+def filter_msg(alert) -> bool:
+    # These are things that recursively happen because Shuffle starts Docker containers
+    skip_rule_ids = ["87924", "87900", "87901", "87902", "87903", "87904", "86001", "86002", "86003", "87932", "80710", "87929",
+            "87928", "5710"]
+    return not alert["rule"]["id"] in skip_rule_ids
+
+    # try:
+    #    if "docker" in alert["rule"]["description"].lower() and "
+    # msg['text'] = alert.get('full_log')
+    # except:
+    #    pass
+    # msg['title'] = alert['rule']['description'] if 'description' in alert['rule'] else "N/A"
+
+
+
+def generate_msg(alert) -> str:
+    if not filter_msg(alert):
+        print("Skipping rule %s" % alert["rule"]["id"])
+        return ""
+
+    level = alert['rule']['level']
+
+    if (level <= 4):
+        severity = 1
+    elif (level >= 5 and level <= 7):
+        severity = 2
+    else:
+        severity = 3
+
+    msg = {'severity': severity, 'pretext': "WAZUH Alert",
+           'title': alert['rule']['description'] if 'description' in alert['rule'] else "N/A",
+           'text': alert.get('full_log'),
+           'rule_id': alert["rule"]["id"],
+           'timestamp': alert["timestamp"],
+           'id': alert['id'], "all_fields": alert}
+
+    return json.dumps(msg)
+
+
+def send_msg(msg: str, url: str):
+    debug("# In send msg")
+    headers = {'content-type': 'application/json', 'Accept-Charset': 'UTF-8'}
+    res = requests.post(url, data=msg, headers=headers, verify=False)
+    debug("# After send msg: %s" % res)
+
+
+if __name__ == "__main__":
+
+    try:
+        # Read arguments
+        bad_arguments = False
+        if len(sys.argv) >= 4:
+            msg = '{0} {1} {2} {3} {4}'.format(
+                now,
+                sys.argv[1],
+                sys.argv[2],
+                sys.argv[3],
+                sys.argv[4] if len(sys.argv) > 4 else '',
+            )
+            # debug_enabled = (len(sys.argv) > 4 and sys.argv[4] == 'debug')
+            debug_enabled = True
```

Comment:
Why is this always enabled?
```suggestion
            debug_enabled = (len(sys.argv) > 4 and sys.argv[4] == 'debug')
```

Reply:
I didn't verify the `debug_enabled` behavior that was in the comment. The suggested change will be tested